examples/custom_search_method/core_search_runner/searcher.yaml

```diff
@@ -0,0 +1,9 @@
+name: core-search-runner
+entrypoint: python3 run_experiment.py
+searcher:
+  metric: validation_error
+  smaller_is_better: true
+  name: single
+  max_length:
```

Comment:
can we exclude max_length here?

Reply:
`config is invalid or incomplete: <config>.searcher: max_length is a required property`  so no, we can't exclude it. Unless there is something I don't know...?